statful/README.md

```diff
@@ -0,0 +1,269 @@
+# Statful client
+
+An Statful client plugin, sending various data to Statful.
+
+## Configuration
+
+```
+config_description = [{'name': 'token',
+                       'type': 'str',
+                       'description': 'Statful API token for authentication.'},
+                      {'name': 'add_custom_tag',
+                       'type': 'str',
+                       'description': 'Add custom tag to statistics'},
+                      {'name': 'batch_size',
+                       'type': 'int',
+                       'description': 'The maximum batch size of grouped metrics to be send to Statful.'}]
+```
+
+The ```token``` parameter is self-explaining. The ```custom_tag``` parameter allows to push a tag key `custom_tag` with a user-input value.
+The ```batch_size``` parameter defines the maximum size of metrics on a batch. 
+
+## Data
+
+All data is send using the [Metrics Ingestion Protocol](https://www.statful.com/docs/metrics-ingestion-protocol.html#Metrics-Ingestion-Protocol):
+
+### Outputs
```

Comment:
Maybe from here to down below use something like `This plugin sends all data that is available on the Gateway metrics bus`. I think you've copied this over from the InfluxDB plugin which also seems to have quite an outdated README.md.

Reply:
Agree with adding that sentence, will update it asap.

I've _forked_ most of the code from InfluxDB plugin, but its README is pretty poor on this part. So I've gone through the codebase and added what metrics are available now.

Reply:
These contents are back from the day when the InfluxDB plugin did the actual collection. In the mean time, we have a metrics bus on which plugins can subscribe/publish, so the InfluxDB plugin (and now the Statful plugin) will subscribe to all metrics, even those from other plugins. So any list in the README.md will be outdated sooner or later, hence it's better to replace it with something more generic.

Reply:
Updated and thanks.